util/create_report_html.py

```diff
@@ -93,11 +84,11 @@ def maybe_get_link(cell, context):
                 namespace = other_prefixes[prefix]
                 url = namespace + local_id
         # IRIs might be in angle brackets
-        iri = re.search(r'((http|https|ftp)://[^ <>]+)', cell)
+        iri = re.search(r'(http://purl.obolibrary.org/obo/[^ <>]+)', cell)
```

Comment:
What if the test is custom?

Reply:
Ok, but if there's a URL inside a text, the text will be a link. I've seen this bug.

Reply:
I trust this change is ok, cant really understand the consequences.